ReText/window.py

```diff
@@ -428,6 +434,26 @@ def dropEvent(e):
 		self.tabWidget.dropEvent = dropEvent
 		self.tabWidget.setTabBarAutoHide(globalSettings.tabBarAutoHide)
 
+	def initDirectoryTree(self, visible, path):
+		if visible:
+			self.fileSystemModel = QFileSystemModel(self.treeView)
+			self.fileSystemModel.setRootPath(path)
```

Comment:
Maybe we should show only supported files (like in `openFile` method), not all files?